VERSIONING.md

```diff
@@ -0,0 +1,31 @@
+# Versioning in the CNB project
+The CNB project versions our [spec components](#spec) and versions our [implementations](#implementations) following seperate schemas, detailed below:
```

Comment:
Do we need to document how libbuildpack is versioned too?

Reply:
Good call will do.